### DISCUSSION TITLE
smart playlist location?

### DISCUSSION THREAD

**Discussion by somedevreally** [jeffvli/feishin#612] [Q&A] [answered]
Hello,

am using Navidrome server, and feishin windows client.
when i create a smart playlist in feishin, i do see it gets added or updated in navidrome. but where is the actual .nsp playlist saved?
i wanted to manually edit/modify/test it.

thanks

**Comment by kgarner7** [comment, marked as answer]
There is no .nsp file; it is stored directly in the database (playlist table).
